tapeout/src/main/scala/transforms/AddSuffixToModuleNames.scala

```diff
@@ -22,9 +21,7 @@ case class ModuleNameSuffixAnnotation(target: CircuitTarget, suffix: String)
 // Verilog black box and therefore can't be renamed.  Since the point is to
 // allow FIRRTL to be linked together using "cat" and ExtModules don't get
 // emitted, this should be safe.
-class AddSuffixToModuleNames extends Transform {
-  def inputForm = LowForm
-  def outputForm = LowForm
+class AddSuffixToModuleNames extends Transform with DependencyAPIMigration {
```

Comment:
How is the ordering of these transforms handled now? I thought we were supposed to add which transforms it needs to run after (i.e. MidToLow or something).